src/RouteConfig.js

```diff
@@ -166,9 +168,10 @@ export default (
   });
 
   (app: any).use((
-    error: string,
+    error: Error,
     request: $Request,
     response: $Response,
+    next: NextFunction, // eslint-disable-line no-unused-vars
```

Comment:
also..I don't understand why, but if we don't pass `next` param here. express doesn't recognize that this middleware is errorHandler middleware...so now it's okey and our multer errors are handled `here`, instead default express errorHandler.